### ISSUE TITLE
Headphone Audio Quality

### ISSUE THREAD

**Issue by bluebearylive** [ExistentialAudio/BlackHole#407]
When I use the 2ch version on my M1 MacBook Air, the audio that comes through my headphones becomes static-y to the point where I can't clearly hear the audio anymore. I've tried the 16ch version but that doesn't work at all, and I have all of the correct settings in my audio-midi set up.

**Comment by bluebearylive** [comment]
Also I should mention, with all of the same settings and set-up, it was working fine with my old (crappy) pair of headphones (which broke), but now with my new higher quality headphones I'm having this audio problem.

**Comment by devinroth** [comment]
It’s an M1 issue.

Devin Roth

**Comment by bluebearylive** [comment]
Ah okay, so I'm guessing there's not much I can do about it then?

**Comment by devinroth** [comment]
Not right now but Like I said. I’ll look into it once I get a chance.

Devin Roth

**Comment by bluebearylive** [comment]
and any idea why it would only work with my old headphones and not my new ones? the only difference is that the new ones have a microphone

**Comment by bluebearylive** [comment]
okay, thank you!

**Comment by devinroth** [comment]
I don’t think you mentioned the mic before. That’s probably the issue. Mics might run on a different sample rate.

Devin Roth

**Comment by bluebearylive** [comment]
Oh! How can I find out what sample rate it runs on?

**Comment by arpart** [comment]
Thanks a lot for your work. And I found the issue with M1 so disturbing that it makes blackhole unusable for me.
As solution I have switched to https://www.gingeraudio.com which works well and smooth with my mini M1
